source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.cpp

```diff
@@ -1,10 +1,10 @@
 #include "CreateMaterialXNodeCmd.h"
 #include "MaterialXNode.h"
-#include "OgsFragment.h"
 #include "MaterialXUtil.h"
 #include "MayaUtil.h"
 #include "Plugin.h"
 
+#include <MaterialXGenOgsXml/OgsFragment.h>
```

Comment:
This file has moved to `MaterialXGenOgsXml`. This is possible since it's actually free from Maya API dependencies. This way, it's covered with CI.

Reply:
This is also possible since we've decided to move the two modules together.